examples/js/loaders/GLTFLoader.js

```diff
@@ -2613,7 +2613,13 @@ THREE.GLTFLoader = ( function () {
 							? new THREE.SkinnedMesh( geometry, material )
 							: new THREE.Mesh( geometry, material );
 
-						if ( mesh.isSkinnedMesh === true ) mesh.normalizeSkinWeights(); // #15319
+						if ( mesh.isSkinnedMesh === true && mesh.geometry.attributes.skinWeight.array instanceof Float32Array ) {
```

Comment:
Instead of checking `skinWeight.array instanceof Float32Array`, could we check `skinWeight.normalized`? I'm worried the first might break for InterleavedBufferAttributes.

Reply:
Good point! Not very familiar with this code; I didn't have models with interleaved attributes to test but it is indeed likely to break.